MAUI/ComboBox/Selection.md

```diff
@@ -64,18 +66,81 @@ The following gif image illustrates the result of the above code:
 
 ![.NET MAUI ComboBox programmatic selection](Images/Selection/ProgramaticSelection.png)
 
+## Multiple selection
+
+The ComboBox allows user to select multiple values from the drop-down list. Selected items will be displayed with a customizable token representation and the users can remove each tokenized item by clicking the close button. 
+
+Token mode supports both editable and non-editable text boxes for selecting items from a data source. The multi-select ComboBox mode can be enabled by setting the SelectionMode property as Multiple.
```

Comment:
Token mode supports editable and non-editable text boxes for selecting items from a data source. The multi-select ComboBox mode can be enabled by setting the SelectionMode property as Multiple.